src/lib/search.ts

```diff
@@ -0,0 +1,14 @@
+const { NODE_ENV, ELASTICSEARCH_URL } = process.env
+const elasticsearch = require("elasticsearch")
+
+const client = new elasticsearch.Client({
```

Comment:
consider adding a check if url is missing or just double check the script exits gracefully in that instance